### PR TITLE
[RU] Added 'advanced/erlang' lesson translation.

### PR DESCRIPTION
**Pull request by brain-geek**

### CODE REVIEW

**Comment by pjhampton** [comment]
You can now alter #220 @brain-geek :)

**Comment by brain-geek** [comment]
@pjhampton Yeah, already done. :)

**Comment by brain-geek** [comment]
@nscyclone Can you please look through this one?

File: ru/lessons/advanced/erlang.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: Взаимодействие с Erlang
+category: advanced
+order: 1
+lang: ru
+---
+
+Одним из достоинств построения языка на инфраструктуре ErlangVM является наличие готовых библиотек. Использование общей ВМ позволяет использовать эти библиотеки, как и стандартную библиотеку Erlang из нашего кода на Elixir. В этом уроке мы рассмотрим как получить доступ: как к функциональности в стандартной библиотеке, так и к сторонним Erlang пакетам.
```
Comment (nscyclone) [inline ru/lessons/advanced/erlang.md]: I think that `пакетам Erlang` sounds better than `Erlang-пакетам`. For example, we usually say `друг человека` and `тарелка борща` but not `человека друг` and `борща тарелка` in everyday speech.
There is also a similar issue on the 52nd line: `Erlang библиотека` → `библиотека Erlang`.

File: ru/lessons/advanced/erlang.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+```erlang
+1> X = 10.
+10
+
+2> X1 = X + 1.
+11
+```
+
+Вот и всё! Использование Erlang из наших приложений Elixir - просто, и это удваивает количество доступных библиотек.
```
Comment (nscyclone) [inline ru/lessons/advanced/erlang.md]: I would translate the last sentence something like: `В наших приложениях Elixir можно с лёгкостью применять Erlang, что удваивает количество доступных библиотек`.

**Comment by nscyclone** [comment]
The translation is wonderful, there are just several small things left to check.
Thanks for a nice work, @brain-geek !

**Comment by brain-geek** [comment]
@nscyclone Thanks for review!

**Pull request merged**